dist/file-preview-card/file-preview-card.css

```diff
@@ -37,125 +43,121 @@
     width: 100%;
 }
 
-.file-preview-card__body--fade {
+.file-preview-card__asset--fade {
     opacity: 0.5;
 }
 
-.file-preview-card__document-icon.icon {
+.file-preview-card__asset.icon {
     height: 40%;
     width: 40%;
 }
 
-.file-preview-card .file-preview-card__menu-action {
-    grid-area: menu-action;
-    justify-self: end;
-    z-index: 1;
-}
-
-.file-preview-card .file-preview-card__menu-action .icon-btn {
+.icon-btn.file-preview-card__action {
+    grid-area: action;
     height: var(--spacing-400);
+    justify-self: end;
     margin: var(--spacing-100);
     width: var(--spacing-400);
+    z-index: 1;
 }
 
-.file-preview-card .file-preview-card__menu-action .icon-btn:not(:hover) {
+.icon-btn.file-preview-card__action:not(:hover) {
     background-color: var(
-        --file-preview-card-menu-action-background-color,
+        --file-preview-card-action-background-color,
         var(--color-background-primary)
     );
 }
 
-.file-preview-card__preview-action {
-    align-self: end;
-    grid-area: label;
-    z-index: 1;
-}
-
-.file-preview-card__preview-action .icon-btn {
+.file-preview-card__label {
     align-items: center;
+    align-self: end;
     background-color: var(
-        --file-preview-card-label-background-color,
+        --file-preview-card-signal-background-color,
         var(--color-background-primary)
     );
+    border-radius: 1em;
     display: flex;
-    flex-direction: row;
-    gap: var(--spacing-50);
-    height: var(--spacing-250);
-    margin: var(--spacing-100);
-    padding: var(--spacing-25) var(--spacing-100);
-    width: max-content;
-}
-
-.file-preview-card__preview-action-text {
-    color: var(
-        --file-preview-card-label-text-color,
-        var(--color-foreground-primary)
-    );
     font-size: var(--font-size-12);
     font-weight: 400;
+    gap: var(--spacing-50);
+    grid-area: signal;
+    margin: var(--spacing-100);
+    padding: var(--spacing-50) var(--spacing-100);
+    z-index: 1;
 }
 
-.file-preview-card__video-icon.icon {
-    height: var(--spacing-150);
-    width: var(--spacing-150);
+.file-preview-card__label > .icon {
+    height: 1em;
```

Comment:
Why switch to `em`? We should probably be using tokens, like before.

Reply:
Yeah, not sure why I changed this. I'll switch back to a token.